### ISSUE TITLE
Feature Possible?

### ISSUE THREAD

**Issue by updawg** [D2R-Gimli/GimliMod#1]
Can the help menu (h button) be replaced with an entirely different image? 

If so listing all runewords, farming areas based on resistances, etc could replace the useless help image.

Just a thought - thanks!

**Comment by artte** [comment]
Old but there are mods with different things like that. Atm I have myself added one but it's Korean so that isn't optimal.
Didn't look into it since it was not just adding a pic in a folder exactly. However should be easy enough to add one if looking into it.

I have a few missing strings and a little weird additions to gimlis version and was mostly for the Mosiac build and tuning down all of the bright lightning etc from it and those other parts was there too so was fine that some was in Korean.

Tldr, very much doable but personally I don't know the specific files for just help screen.
Gimli might have some idea. It think an update is not really prioritized for him. Need to get a hold of him and talk to him.
